homework/index.js

```diff
@@ -30,14 +32,145 @@
     return elem;
   }
 
+  function openNewTab(url) {
+    window.open(url, '_blank');
+  }
+
+  function bindContributions(contributorsUrl) {
+    fetchJSON(contributorsUrl, (error, response) => {
+      const repoInfoContainer = document.getElementById('repo-info-container');
+      const contributorsDiv = createAndAppend('div', repoInfoContainer, {
+        class: 'contributors-info',
+      });
+      if (error) {
+        createAndAppend('div', contributorsDiv, {
+          text: error.message,
+          class: 'alert-error',
+        });
+        return;
+      }
+
+      const div = createAndAppend('div', contributorsDiv);
+      createAndAppend('h4', div, { text: 'Contributions' });
+      createAndAppend('hr', div);
+      const table = createAndAppend('div', div, { class: 'grid' });
+      const contributors = response;
+      for (let index = 0; index < contributors.length; index++) {
```

Comment:
While an old-style `for` loop will work fine, many developers, including myself, would prefer a `.forEach()` if you just want to loop through the elements of an array.

Reply:
done